packages/cf-style-provider/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I would like to but we depend on lerna here. The yarn support is [experimental](https://github.com/lerna/lerna/issues/371), let's wait when it's more stable.

> We don't want failed CI builds every time fela upgrades

It should not be *every* time.